connectors/sqs/src/main/java/io/camunda/connector/inbound/SqsQueueConsumer.java

```diff
@@ -38,28 +39,37 @@ public SqsQueueConsumer(
   @Override
   public void run() {
     LOGGER.info("Started SQS consumer for queue {}", properties.getQueue().getUrl());
+
     final ReceiveMessageRequest receiveMessageRequest = createReceiveMessageRequest();
     ReceiveMessageResult receiveMessageResult;
     do {
       receiveMessageResult = sqsClient.receiveMessage(receiveMessageRequest);
       List<Message> messages = receiveMessageResult.getMessages();
       for (Message message : messages) {
-        InboundConnectorResult<?> correlate =
-            context.correlate(MessageMapper.toSqsInboundMessage(message));
-        if (correlate.isActivated()) {
+        try {
+          correlate(message);
           sqsClient.deleteMessage(properties.getQueue().getUrl(), message.getReceiptHandle());
```

Comment:
@Oleksiivanov We dont delete messages that failed correlation because of an expception but we delete them in case of failed acitvation. If you read the comments from @chillleader you can see that we dont requeue messages that were not activated:
> NACK (no requeue) - inbound event not correlated
getChannel().basicReject(envelope.getDeliveryTag(), false);

In the case of a failed correlation because of an exeption we do requeue: 
> NACK (requeue) - failed to correlate event", e);
getChannel().basicReject(envelope.getDeliveryTag(), true);